tests/src/JIT/Regression/JitBlue/vso_176584/vso_176584.cs

```diff
@@ -0,0 +1,35 @@
+static class MicrosftNETBug
+{
+
+  struct S
+  {
+    public readonly ushort D;
+    
+    public S(ushort d) { D = d; } 
+    public static bool operator >(S a, S b) { return a.D > b.D; }
+    public static bool operator <(S a, S b) { return a.D < b.D; }
+  }
+
+
+  class C
+  {
+    public S D;
+    
+    public C(ushort d) { D = new S(d); }
+  }
+
+ 
+
+  static void Main()
+  {
+
+    Console.WriteLine(".NET 4.6.01055 bug repro: second line incorrectly outputs False (only) when code is optimized");
+    
+    C c3 = new C(0);
+    C c1 = new C(256 * 128 - 1);
+    C c2 = new C(256 * 128);
+
+    Console.WriteLine(c1.D > c3.D);  
+    Console.WriteLine(c2.D > c3.D);            /* Gives false when optimization is enabled which is not correct */
+  }
```

Comment:
I thought tests were supposed to return 100 to indicate success and others indicate failure.  Is there some other test harness magic making this pass that I'm not aware of?  (Wouldn't be surprised)  But if not, I think we may need an `if` to control the result.  Obviously may be easier or harder depending on the sensitivity of the bug to specific code patterns.